koza/model/config/source_config.py

```diff
@@ -151,13 +151,11 @@ class SourceFileConfig:
     name: str
     files: List[Union[str, Path]]
     format: FormatType = FormatType.csv
-    path: Path = None
```

Comment:
i had a mental note to come back to this, did it turn out we didn't need it?  I was wondering if I broke something with the koza testing PR

Reply:
I wasn't sure what this one was supposed to do, and I couldn't find it being used.

Reply:
We added it in here - https://github.com/monarch-initiative/koza/pull/28/files#diff-91315ee69713516b3cf1a680459d360724e814af1885d65b8e30693970eb7cedR154

Reply:
🤦‍♂️

Reply:
it might not be needed, I was thinking the path could be picked up from the transform_code

Reply:
I'll leave it in for now, but since we don't need anyone to put it in a config file, I won't put it into the documentation